build.gradle

```diff
@@ -27,18 +27,18 @@ ext {
     minSdkVersion = 14
     compileSdkVersion = 26
```

Comment:
@trr-amsiq @tristanrichard guys, you forgot to update `compileSdkVersion` to `27`, that's why Travis build still fails.